MultimodalRAGWithVideos/docker_image_build/build.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+services:
+  multimodalragwithvideos:
+    build:
+      args:
+        http_proxy: ${http_proxy}
+        https_proxy: ${https_proxy}
+        no_proxy: ${no_proxy}
+      dockerfile: ./Dockerfile
+    image: ${REGISTRY:-opea}/multimodalragwithvideos:${TAG:-latest}
+  # multimodalragwithvideos-ui:
+  #   build:
+  #     context: ui
+  #     dockerfile: ./ui/Dockerfile
+  #   extends: multimodalragwithvideos
+  #   image: ${REGISTRY:-opea}/multimodalragwithvideos-ui-ui:${TAG:-latest}
+  bridgetower-embedding-server:
```

Comment:
Please keep the service name the same as the image name, `bridgetower-embedding-server` -> `bridgetower-embedder`

Reply:
@chensuyue Thanks for the feedback. I have corrected all service names to be the same with their corresponding image names. Please check again and click resolve conversation. Thanks a lot